draft-ietf-quic-transport.md

```diff
@@ -1834,26 +1834,31 @@ congestion in the network by setting a codepoint in the IP header of a packet
 instead of dropping it.  Endpoints react to congestion by reducing their sending
 rate in response, as described in {{QUIC-RECOVERY}}.
 
-To use ECN, QUIC endpoints first determine whether a path and peer support ECN
-marking. Verifying the path occurs at the beginning of a connection and when the
+To use ECN, QUIC endpoints first determine whether a path supports ECN
+marking and the peer is able to access the ECN codepoint in the IP header. 
+A network path regarded as not supporting ECN, if ECN marked packets get dropped
+or ECN marking are rewritten on the path.
+Verifying the path occurs at the beginning of a connection and when the
```

Comment:
Passive voice.  "An endpoint verifies every path, both during connection establishment and when migrating to a new path (see {{migration}})."